executor/executor.cc

```diff
@@ -300,6 +300,7 @@ struct handshake_req {
 	uint64 magic;
 	uint64 flags; // env flags
 	uint64 pid;
+	int sandbox_arg;
```

Comment:
Maybe also let it be uint64 here? Just to avoid problems in the future due to struct padding.

Reply:
+1